scenarios/proxy.grpc.benchmarks.yml

```diff
@@ -0,0 +1,130 @@
+﻿variables:
+    path: /
+    serverScheme: https # http, https
+    downstreamScheme: https # http, https
+    serverPort: 8080
+    downstreamPort: 5000
+    connections: 1
+    streams: 100
+    grpcClientType: GrpcNetClient
+    scenario: serverstreaming
+    protocol: h2
+    warmup: 5
+    duration: 5
+    logLevel: none
+    requestSize: 0
+    responseSize: 0
+    enableCertAuth: false
+    latency: false
+
+jobs:
+  grpcDotNetClient:
+    source:
+      repository: https://github.com/grpc/grpc-dotnet.git
+      branchOrCommit: master
+      project: perf/benchmarkapps/GrpcClient/GrpcClient.csproj
+    readyStateText: gRPC Client
+    waitForExit: true
+    arguments: "-c {{connections}} --streams {{streams}} -d {{duration}} -p {{protocol}} -s {{scenario}} -u {{serverScheme}}://{{serverAddress}}:{{serverPort}}{{path}} -w {{warmup}} --grpcClientType {{grpcClientType}} --requestSize {{requestSize}} --responseSize {{responseSize}} --enableCertAuth {{enableCertAuth}} --latency {{latency}} {% if logLevel != 'none' %} --logLevel {{ logLevel }} {% endif %}"
+  grpcAspNetCoreServer:
+    source:
+      repository: https://github.com/grpc/grpc-dotnet.git
+      branchOrCommit: master
+      project: perf/benchmarkapps/GrpcAspNetCoreServer/GrpcAspNetCoreServer.csproj
+    readyStateText: Application started
+    waitForExit: false
+    arguments: "--LogLevel {{logLevel}} --enableCertAuth {{enableCertAuth}} --protocol {{protocol}}"
+  yarp:
+    source:
+      repository: https://github.com/microsoft/reverse-proxy.git
+      branchOrCommit: main
+      project: samples/BenchmarkApp/BenchmarkApp.csproj
+    readyStateText: Application started.
+    arguments: '--urls {{serverScheme}}://*:8080 --clusterUrls {{downstreamScheme}}://{{downstreamAddress}}:{{downstreamPort}}'
+scenarios:
+  proxy-yarp-grpc:
+    downstream:
+      job: grpcAspNetCoreServer
+    application:
+      job: yarp
+    load:
+      job: grpcDotNetClient
+
+profiles:
```

Comment:
Would it work if you imported this file instead: https://github.com/aspnet/Benchmarks/blob/master/scenarios/aspnet.profiles.yml?raw=true

In the `imports` sections. I am trying to centralize this list of servers.

Reply:
I tried that at first, but then I have to define the variables (streams, protocol, etc.) in each of the scenarios. I will add more proxies besides YARP and want to avoid variables duplication.